src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.String.cs

```diff
@@ -6,6 +6,21 @@ namespace System.Text.Json.Serialization
 {
     public static partial class JsonSerializer
     {
+        /// <summary>
+        /// Read the UTF-16 encoded JSON <see cref="System.String"/> and return the converted value.
+        /// </summary>
+        /// <returns>The converted value.</returns>
+        /// <param name="json">The UTF-16 encoded <see cref="System.String"/>.</param>
+        /// <param name="options">The <see cref="JsonSerializerOptions"/> used to read and convert the JSON.</param>
+        /// <exception cref="System.ArgumentNullException">
+        /// Thrown if <paramref name="json"/> is null.
+        /// </exception>
+        /// <exception cref="JsonReaderException">
+        /// Thrown when the JSON is invalid or when <typeparamref name="TValue"/> is not compatible with the JSON.
+        /// </exception>
+        /// <remarks>Using a UTF-16 <see cref="System.String"/> is not as efficient as using the
+        /// UTF-8 methods since the implementation natively uses UTF-8.
```

Comment:
**Question:** Would it be more performant for the user to first call `Encoding.UTF8.GetBytes(string json)` and pass it as a `ReadOnlypan<byte>` rather than just passing the string here and let us do it? Or is it about the same.

Reply:
It would be the same assuming they provide the option to validate (like we do).